frontend/.gitignore

```diff
@@ -32,4 +32,6 @@ next-env.d.ts
 
 *storybook.log
 
-.env.test
\ No newline at end of file
+.env.test
+# Sentry Config File
+.env.sentry-build-plugin
```

Comment:
Are you sure about this path?
Is it really a config file? 🤔

Reply:
This was put there automatically, yes, but we don't need it :)